library/src/main/scala/scala/scalajs/runtime/LinkingInfo.scala

```diff
@@ -15,6 +15,8 @@ package scala.scalajs.runtime
 import scala.scalajs.js
 
 /** Information about link-time configuration of Scala.js. */
+@deprecated("Use scala.scalajs.LinkingInfo instead." +
+    "For fileLevelThis, use scala.scalajs.js.special.fileLevelThis.", "1.17.0")
```

Comment:
```suggestion
@deprecated(
    "Use scala.scalajs.LinkingInfo instead. " +
    "For fileLevelThis, use scala.scalajs.js.special.fileLevelThis.",
    since = "1.17.0")
```